packages/components/src/components/install-guide/ProjectPickerRequirements.vue

```diff
@@ -0,0 +1,275 @@
+<template>
+  <div v-if="project">
```

Comment:
It seems like the default constructor for the prop isn't working, because `projects` is `undefined` when a project isn't selected in the table. But I'll change this so that we're not relying on the default constructor failing to work as expected.